store/src/db/sql.rs

```diff
@@ -358,8 +359,9 @@ pub fn select_notes(conn: &mut Connection) -> Result<Vec<Note>> {
                 batch_index: row.get(1)?,
                 note_index: row.get(2)?,
                 note_id,
-                sender: column_value_as_u64(row, 4)?,
-                tag: column_value_as_u64(row, 5)?,
+                note_type: row.get::<_, u8>(4)?.try_into()?,
```

Comment:
These don't look too good versus a specific function, but I think it's good enough.